lib/node/p2p.js

```diff
@@ -16,11 +16,13 @@ const tcp = new TcpTransport({
 });
 
 tcp.bootstrap().then((nodes) => {
-  console.log('[check nodes]', nodes);
-  const remoteHost = nodes[0][0];
-  const remotePort = nodes[0][1];
-  console.log('Connecting to', nodes[0]);
-  tcp.connect(remotePort, remoteHost);
+  try {
+    const remoteHost = nodes[0][0];
+    const remotePort = nodes[0][1];
+    tcp.connect(remotePort, remoteHost);
+  } catch (error) {
```

Comment:
@vikas-ranadev let's only use either .then().catch() chaining for catching errors or try{}catch(){} blocks. Since this has been removed hence no action to be taken.

Reply:
Resolved in this commit https://github.com/imthe-1/multi-dimension-identity-protocol/pull/4/commits/fd8ec0215e6d6982da0ef543610aa61e806030a6